src/main/webapp/WEB-INF/classes/spring/fcrepo4-config.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As we begin to move away from the "Fedora 4" branding, it would be good if we name this file `fcrepo-config.xml` instead of `fcrepo4-config.xml`.
See: https://github.com/fcrepo4-exts/fcrepo-webapp-plus/pull/55#discussion_r99419605